scripts/issue/474/upgrade_arb_crv_univ3.py

```diff
@@ -0,0 +1,128 @@
+from decimal import Decimal
+
+from brownie import accounts, web3, interface, chain
+from rich.console import Console
+
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import ADDRESSES_ARBITRUM
+from helpers.addresses import registry
+
+console = Console()
+
+DEV_MULTI = registry.arbitrum.badger_wallets.dev_multisig
+PROXY_ADMIN = registry.arbitrum.proxyAdminDev
+
+TRICRYPTO_STRATEGY = ADDRESSES_ARBITRUM["strategies"]["native.tricrypto"]
+TRICRYPTO_LOGIC = ADDRESSES_ARBITRUM["logic"]["native.tricrypto"]
+TRICRYPTO_VAULT = ADDRESSES_ARBITRUM["sett_vaults"]["bcrvTricrypto"]
+
+RENCRV_STRATEGY = ADDRESSES_ARBITRUM["strategies"]["native.renCrv"]
+RENCRV_LOGIC = ADDRESSES_ARBITRUM["logic"]["native.renCrv"]
+RENCRV_VAULT = ADDRESSES_ARBITRUM["sett_vaults"]["bcrvRenBTC"]
+
+UNIV3_ROUTER = "0xE592427A0AEce92De3Edee1F18E0157C05861564"
```

Comment:
suggest to include this in the address book instead (it already is for mainnet; `r.uniswap.routerV3`)